### PR TITLE
[MALI- 1161] - Fix Preview-codes backslash issue

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
QR Code feature is not working in PROD and testflight builds. Pretty strange that it happens only in Prod builds.
Removing any backslash and \\ in preview token that could break the preview feature in the PROD. We suspect that this could be the reason for the issue.
Preview tokens adds a extra / sometimes, this could help avoid the situation.

## Links
[MALI-1161](https://jira.rakuten-it.com/jira/browse/MALI-1161)

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Sources/Classes/core/API/PreviewMiniappAPI.swift [hunk 1]
```diff
@@ -15,9 +15,10 @@ internal class PreviewMiniappAPI {
     }
 
     private func getMiniAppInfo(using token: String) -> URL? {
+        let previewToken = token.replacingOccurrences(of: "\\\\/", with: "", options: .regularExpression, range: nil)
```
Comment (Climbatize) [inline Sources/Classes/core/API/PreviewMiniappAPI.swift:18]: Do you really need a regex for that, seems over-engineering to me ^^

**Review by Climbatize — APPROVED** [review]
LGTM, good finding!

File: Sources/Classes/core/API/PreviewMiniappAPI.swift (hunk 1 above)
Comment (rleojoseph) [reply Sources/Classes/core/API/PreviewMiniappAPI.swift:18]: Yeah, I was thinking of an easy approach but sometimes what if we get too many / in the preview code? in that case regex would be easy solution

**Review by laubengaier — APPROVED** [review]
lgtm!

**Pull request merged**